src/components/NewEntry.spec.js

```diff
@@ -0,0 +1,134 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import NewEntry from './NewEntry'
+import userEvent from '@testing-library/user-event'
+
+describe('NewEntry', () => {
+  it('has radio input fields with the expected text', () => {
+    render(<NewEntry />)
+
+    const firstInputField = screen.getByDisplayValue('weiß')
+    const secondInputField = screen.getByDisplayValue('einfarbig')
+    const thirdInputField = screen.getByDisplayValue('Baumwolle')
+    const forthInputField = screen.getByDisplayValue('Figurbetont')
+
+    expect(firstInputField).toBeInTheDocument()
+    expect(secondInputField).toBeInTheDocument()
+    expect(thirdInputField).toBeInTheDocument()
+    expect(forthInputField).toBeInTheDocument()
+  })
+
+  it('has two input fields with the expected label-texts', () => {
+    render(<NewEntry />)
+
+    const labelText1 = screen.getByLabelText(
+      'Gib einen Namen für dein Kleidungsstück ein:',
+    )
+
+    const labelText2 = screen.getByLabelText(
+      'Zu welcher Kategorie soll das Kleidungsstück hinzugefügt werden?',
+    )
+
+    expect(labelText1).toBeInTheDocument()
+    expect(labelText2).toBeInTheDocument()
+  })
+
+  it('has 2 input fields of the textbox type', () => {
+    render(<NewEntry />)
+
+    const inputField = screen.getAllByRole('textbox')
+
+    expect(inputField).toHaveLength(2)
+  })
+
+  it('has 32 input fields of the radio type', () => {
+    render(<NewEntry />)
+
+    const inputField = screen.getAllByRole('radio')
+
+    expect(inputField).toHaveLength(32)
+  })
+
+  xit('has 34 input fields which are required', () => {
+    render(<NewEntry />)
+
+    const inputFieldType1 = screen.getAllByRole('input')
+    const inputFieldType2 = screen.getAllByRole('radio')
+
+    expect(inputFieldType1).toBeRequired()
+    expect(inputFieldType2).toBeRequired() // funktiniert nich bei Arrays
```

Comment:
I would delete the comment.

Reply:
Yes, it's a reminder for me, because I did not figure out yet, how to write the test that it will work. Have skipped the test at the moment (used the 'xit' for it)